README.rst

```diff
@@ -118,10 +118,35 @@ Following most recent variants are available:
 
 .. list-table::
     :header-rows: 1
+    :stub-columns: 1
 
-    * - Cowbird
-    * - pavics/cowbird:0.4.0
-    * - pavics/cowbird:latest
+    * - Version
+      - Cowbird Base
+      - Cowbird Worker
+      - Cowbird Web Service
+    * - Most Recent Release
+      - |cowbird_tag_base|_
+      - |cowbird_tag_worker|_
+      - |cowbird_tag_websvc|_
+    * - Latest Commit
+      - |cowbird_latest_base|_
+      - |cowbird_latest_worker|_
+      - |cowbird_latest_websvc|_
+
+
+.. |cowbird_tag_base| replace:: pavics/cowbird:0.4.0
+.. _cowbird_tag_base: https://hub.docker.com/r/pavics/magpie/tags?page=1&ordering=last_updated&name=0.4.0
+.. |cowbird_tag_worker| replace:: pavics/cowbird:0.4.0-worker
+.. _cowbird_tag_worker: https://hub.docker.com/r/pavics/magpie/tags?page=1&ordering=last_updated&name=0.4.0-worker
+.. |cowbird_tag_websvc| replace:: pavics/cowbird:0.4.0-webservice
+.. _cowbird_tag_websvc: https://hub.docker.com/r/pavics/magpie/tags?page=1&ordering=last_updated&name=0.4.0-webservice
+
+.. |cowbird_latest_base| replace:: pavics/cowbird:latest
+.. _cowbird_latest_base: https://hub.docker.com/r/pavics/magpie/tags?page=1&ordering=last_updated&name=latest
+.. |cowbird_latest_worker| replace:: pavics/cowbird:latest-worker
+.. _cowbird_latest_worker: https://hub.docker.com/r/pavics/magpie/tags?page=1&ordering=last_updated&name=latest-worker
+.. |cowbird_latest_websvc| replace:: pavics/cowbird:latest-webservice
+.. _cowbird_latest_websvc: https://hub.docker.com/r/pavics/magpie/tags?page=1&ordering=last_updated&name=latest-webservice
```

Comment:
Links lead to magpie instead of cowbird

Reply:
oh, copy-paste problem
nicely spotted 👍